intermol/lammps/lammps_parser.py

```diff
@@ -1203,9 +1199,13 @@ def write(self, unit_set='real'):
 
             # non-bonded
             if atom_charges:
-                f.write('pair_style lj/cut/coul/cut 9.99999 19.99999\n')  # TODO: match mdp
```

Comment:
I'll play around with these settings (plus the gromacs_vacuum, onepoint.cfg settings.